test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -67,7 +67,13 @@ var _ = SIGDescribe("[HPA] Horizontal pod autoscaling (scale resource: CPU)", fu
 	})
 
 	SIGDescribe("ReplicationController light", func() {
-		ginkgo.It("Should scale from 1 pod to 2 pods", func() {
+		/*
+		  Release : v1.16
+		  Testname: Up-scale pod replicas and resources
+		  Description: Pod replicas MUST increase from 1 pod to 2 pods.
+		  Pod CPU request will increase to 50%.
```

Comment:
It should be: Average Pod CPU utilization will drop to lower than 50%

Utilization is usage/request
The test starts with 1 pod, 150 mCpu usage, 200mcpu request - that gives us 75% utilization, while target is 250.
Autoscaler will add second pod. Then we have 150 mCpu total usage, 400 mCPU total request which is <50% utilization.